packages/next-swc/crates/next-dev-tests/tests/integration.rs

```diff
@@ -518,10 +528,18 @@ impl IssueReporter for TestIssueReporter {
         captured_issues: TransientInstance<ReadRef<CapturedIssues>>,
         _source: TransientValue<RawVc>,
     ) -> Result<BoolVc> {
+        let log_options = LogOptions {
+            current_dir: PathBuf::new(),
+            project_dir: PathBuf::new(),
+            show_all: true,
+            log_detail: true,
+            log_level: IssueSeverity::Info,
+        };
         let issue_tx = self.issue_tx.get_untracked().clone();
         for (issue, path) in captured_issues.iter_with_shortest_path() {
             let plain = NormalizedIssue(issue).cell().as_issue().into_plain(path);
             issue_tx.send((plain.await?, plain.dbg().await?))?;
+            println!("{}", format_issue(&*plain.await?, None, &log_options));
```

Comment:
Intentional?

Reply:
yep, otherwise it doesn't show issues when debugging tests...